test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
Just did this — https://github.com/averyvery/university/commit/b2d95b9fb61e4a200dd067624bda668df4c86c70

At least for the moment, I can't think of any shared resources these would have besides a port, but then, this is my first node server....